### PR TITLE
fix and improve mobile UX

### PR DESCRIPTION
**Pull request by szaimen**
Fix https://github.com/nextcloud/spreed/issues/1937

Signed-off-by: szaimen <szaimen@e.mail.de>

<details>
<summary>For my own testing</summary>

```
docker run -it ^
-e TRUSTED_DOMAIN=192.168.178.155 ^
-e TALK_BRANCH=enh/1937/mobile ^
--name nextcloud-easy-test ^
-p 8443:443 ^
--volume="nextcloud_easy_test_npm_cache_volume:/var/www/.npm" ^
ghcr.io/szaimen/nextcloud-easy-test:latest
```

</details>

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -67,6 +67,7 @@ import ConversationSettingsDialog from './components/ConversationSettings/Conver
 import '@nextcloud/dialogs/styles/toast.scss'
 import { CONVERSATION } from './constants.js'
 import DeviceChecker from './components/DeviceChecker/DeviceChecker.vue'
+import isMobile from '@nextcloud/vue/dist/Mixins/isMobile'
```
Comment (nickvergessen) [inline src/App.vue:70]:
```suggestion
import isMobile from '@nextcloud/vue/dist/Mixins/isMobile.js'
```

File: src/App.vue [hunk 2]
```diff
@@ -375,6 +377,12 @@ export default {
 		} else if (BrowserStorage.getItem('sidebarOpen') === 'true') {
 			this.$store.dispatch('showSidebar')
 		}
+		if (this.isMobile) {
+			await this.$nextTick()
+			emit('toggle-navigation', {
+				open: true,
```
Comment (marcoambrosini) [inline src/App.vue:383]: innit `false` here??

Comment (szaimen) [reply src/App.vue:383]: no, we want the chatview to load on mobile automatically.

Comment (nickvergessen) [reply src/App.vue:383]: But not when you opened a URL of a conversatino already

Comment (szaimen) [reply src/App.vue:383]: fixed

**Review by marcoambrosini — APPROVED** [review]

**Pull request merged**